echopype/convert/set_groups_ek60.py

```diff
@@ -241,6 +241,11 @@ def set_platform(self) -> xr.Dataset:
         time1, msg_type, lat, lon = self._extract_NMEA_latlon()
 
         # NMEA dataset: variables filled with nan if do not exist
+        platform_dict = {
+            "platform_name": self.ui_param["platform_name"],
+            "platform_type": self.ui_param["platform_type"],
+            "platform_code_ICES": self.ui_param["platform_code_ICES"],
+        }
```

Comment:
For this and line 279-285: What do you think we just remove this TODO, and also remove the said EK80 capability? In the echopype examples notebook we showed how to modify it directly on the EchoData object, so maybe we should just remove the capability to put it in as part of conversion all together?
https://github.com/OSOceanAcoustics/echopype/blob/8aa4d9f77d7f09c269c3441d22920b3985b5a0de/echopype/convert/set_groups_ek60.py#L279-L285

Reply:
> For this and line 279-285: What do you think we just remove this TODO, and also remove the said EK80 capability? In the echopype examples notebook we showed how to modify it directly on the EchoData object, so maybe we should just remove the capability to put it in as part of conversion all together?

I confess I'd never bothered to see what `ui_param` is! If I get that right (from a quick code scan), it allows passing certain parameter values during the conversion process, on `open_raw`, and some of those parameters are used for setting those global `param_*` attributes. If that's correct, I wasn't aware that was there! Let's talk about this today to make sure I understand. 

I'm all for removing that long TODO comment, and likewise removing the EK80 water_level updating capability that's based on `self.ui_param['water_level']`.

Reply:
I've removed that ek60 comment block, and also removed the `self.ui_param` statements that referred to the Platform global `param_*` attributes in all 3 sensors, while I was at it.

I'll create a new issue about removing the `ui_param` property.

Reply:
Opened new issue #1059